_rules/SC4-1-1-unique-id.md

```diff
@@ -20,7 +20,7 @@ authors:
 
 ### Applicability
 
-Any `id` attribute specified on an HTML or SVG element.
+Any [non-empty](#non-empty) `id` attribute specified on an HTML or SVG element.
```

Comment:
Non-empty isn't the right definition here. I can't really tell what you were trying to do here because you didn't add any test cases. But I doubt `id="-"` should be ignored by this, which is what "non-empty" does.

Reply:
We discussed whether it would make sense to split the "non-empty" definition out into two different definitions: 

1. Non-empty: A string of characters is considered non-empty if the string contains characters that are not space characters. (or even have the definition be of "empty" instead)

2. Textual content: A string of characters is considered textual content if it contains one or more characters from the letter or number categories of Unicode. This definition applies to the string as a whole, regardless of any other characters the string might contain.

I think this would ease some of the confusion around what the "non-empty" definition means.

I know this change would require us to go through already published rules and see which of the definitions it should be using. But I am guessing that such a second look could be beneficial.

Reply:
@jkodu are you going to pick that up? What's the plan for this?